build_deploy.sh

```diff
@@ -6,7 +6,7 @@ set -o pipefail
 
```

Comment:
I'd remove this first in a MR in which you cleanup all the previous build system scripts